README.md

```diff
@@ -248,12 +248,20 @@ There are 2 types of caches supported: an in memory one using [go-cache](https:/
 Set `cache` option to true to use a cache (defaults to false when missing) and `cache_type` to set the type of the cache. By default the plugin will use `go-cache` unless explicitly told to use Redis.
 Set `cache_reset` to flush the cache on mosquitto startup (**hydrating `go-cache` on startup is not yet supported**).
 
+
+**Update v1.2:**
```

Comment:
What do you think about adding a warning to ensure people are aware of what can happen if `cache_refresh true` is set? Maybe something like: 

Setting `cache_refresh true` in combination with continuously publishing/receiving messages more frequently than `auth_cache_seconds` and `acl_cache_seconds` can lead to cache entries getting "stuck" in the cache even though they aren't present in the backend any more.

Reply:
Thanks for the suggestion, I added a small warning about it.